webapp/src/components/training/TrainingInformation.vue

```diff
@@ -258,14 +258,14 @@ const lossSeries = computed(() =>
       },
     ),
 );
-
+const darkMode = localStorage.getItem("theme") === "dark";
 const commonChartsOptions = {
```

Comment:
Oh also can you make the axes tick labels lighter? We can't read the values very well in the dark theme
![Screenshot 2024-10-02 at 15 00 27](https://github.com/user-attachments/assets/6fd2f5b4-0708-47f5-a60c-406bd78a56f5)